tests/tutorials/AppiumNativeTest.php

```diff
@@ -57,4 +63,7 @@ public function demo()
 }
 
 $test = new HelloWorld();
-$test->demo();
\ No newline at end of file
+try {
```

Comment:
why you've added the catching the exception?

Reply:
Because of Code Analysis warning.